include/seqan3/argument_parser/detail/format_base.hpp

```diff
@@ -193,6 +253,25 @@ class format_base
 
         return tmp;
     }
+
+    /*!\brief Appends a double dash to a long identifier and returns it.
+    * \param[in] long_id The name of the long identifier.
+    * \returns The input long name prepended with a double dash.
+    */
+    std::string prepend_dash(std::string const & long_id)
+    {
+        return ("--" + long_id);
+    }
+
+    /*!\brief Appends a double dash to a short identifier and returns it.
```

Comment:
it seems you moved this here from format_parse? What was the reason?
And can you fix the documentation. Only one dash is added to the short identifier.

Reply:
I moved the function from `seqan3::detail::format_parse` such that it is visible from `seqan3::detail::format_ctd` without inheriting from `seqan3::detail::format_parse`, but if it is a problem I can for sure revert this change.